packages/querygrid/src/components/search/SearchResultsPanel.tsx

```diff
@@ -56,7 +56,11 @@ export class SearchResultsPanel extends React.Component<Props, any> {
         const results = model ? model.getIn(['entities', 'hits']) : undefined;
 
         if (!this.isLoading() && results !== undefined) {
-            const data = results.filter((result) => result.has('data'));
+            // result.has('data') is <=20.1 compatible way to check for sample search results TODO remove post 20.1
```

Comment:
I think we need to use` result.has('data')` to make this work with 19.3 patches that may need to happen.